### PR TITLE
Cards component

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: Components/cards/cards.css [hunk 1]
```diff
@@ -140,23 +140,43 @@ body {
...
-    padding: 0.3rem 0.5rem;
-    border-radius: 3px;
-    font-size: 0.8rem; */
+/* card with badge */
+
+.card-withBadge {
+  position: absolute;
+  top: 10px;
+  left: 9px;
+  background-color: #C9CCD5;
```
Comment (arjun-computer-geek) [inline Components/cards/cards.css:154]: use css variable instead of actual value

Comment (abhisheksahani-nova) [reply Components/cards/cards.css:154]: I will , once I start with documentation site , because for now I havent decided my color palette

**Comment by mikesch803** [comment]
LGTM

**Review by mikesch803 — APPROVED** [review]

**Review by VLeads — APPROVED** [review]
Code looks good. Go ahead !!

File: Components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  height: 20px;
+}
+
+/* card text over media */
+
+.card-text-overMedia {
+  position: relative;
+}
+
+.heading-subHeading-overMedia-container {
```
Comment (SatwikaKacham) [inline Components/cards/cards.css:74]: i think there is no class with .heading-subHeading-overMedia-container  if not needed u can remove this

File: Components/cards/cards.css [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+.card-subHeading {
+  margin-left: 12px;
+  letter-spacing: 0.03rem;
+  color: #6a6e6e;
+  font-weight: 500;
+}
+
+.card-description {
+  margin-left: 12px;
```
Comment (SatwikaKacham) [inline Components/cards/cards.css:38]: using rem is preferred i think that makes ur whole code consistent

File: Components/cards/cards.html [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+              <span class="icon"><i class="fa fa-shopping-cart"></i></span> Add
+              to Cart
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <div class="container">
```
Comment (SatwikaKacham) [inline Components/cards/cards.html:131]: it would be better if u specify which container it is like card container

**Review by SatwikaKacham — APPROVED** [review]
Everything is perfect.suggested small changes good to merge.

File: Components/cards/cards.css (hunk 2 above)
Comment (abhisheksahani-nova) [reply Components/cards/cards.css:74]: Yaah actually its for different version of card , which I will be using in future, currently I am not using it , thats why you cant see it in html file

File: Components/cards/cards.css (hunk 3 above)
Comment (abhisheksahani-nova) [reply Components/cards/cards.css:38]: yess

File: Components/cards/cards.html (hunk 4 above)
Comment (abhisheksahani-nova) [reply Components/cards/cards.html:131]: okay i will do that

**Pull request merged**